tasklib/serializing.py

```diff
@@ -38,16 +38,22 @@ class SerializingObject(object):
       - If validation or normalization fails, normalizer is expected
         to raise ValueError.
     """
+    TIMESTAMP_FIELDS = {'entry', 'modified', 'start', 'end', 'due',
+                        'scheduled', 'until', 'wait'}
 
     def __init__(self, backend):
         self.backend = backend
 
     def _deserialize(self, key, value):
+        if key in self.TIMESTAMP_FIELDS:
+            return self.timestamp_deserializer(value)
         hydrate_func = getattr(self, 'deserialize_{0}'.format(key),
```

Comment:
This is where the `deserialize_*`  methods are called from (same for `_serialize` and `_normalize` below), so they are actually used.

Reply:
The condition above:

```python
if key in self.TIMESTAMP_FIELDS:
       return self.timestamp_deserializer(value)
```

handles all the removed members, so the line:

```python
hydrate_func = getattr(self, 'deserialize_{0}'.format(key),
```

won't be reached. 

My goal is to remove redundancy in the code, as all the members I deleted have the exact same body.  I plan to use the library with django for a web frontend.

I understand that this is mostly a matter of style so no big deal if it isn't worth merging. Is there anything you want to have or change in particular?

Reply:
Oh, I see that the idea is to reduce redundancy here. I'd rather keep the separate methods to allow easier extensibility, though I do agree that it's a little repetitive 😄